### PR TITLE
Copy updates

### PR DESCRIPTION
**Pull request by PatFawbertMills**

### CODE REVIEW

File: src/sites/cclw/constants/faqs.tsx [hunk 1]
```diff
@@ -41,25 +41,29 @@ export const FAQS: TFAQ[] = [
...
         <p>
           This database originates from a collaboration between the Grantham Research Institute and GLOBE International on a series of Climate
           Legislation Studies. Since then, Climate Change Laws of the World has transformed into one of the Grantham Research Institute’s core
           strategic projects, underpinning much of the Institute’s work on our governance and legislation theme.
         </p>
         <p>
-          The database is now powered by new technology built by our partners{" "}
-          <ExternalLink url="https://climatepolicyradar.org/">Climate Policy Radar</ExternalLink>. This includes a series of features and
-          functionality to make it easier to explore the data.
+          The database is now powered and operated by our partners{" "}
```
Comment (diversemix) [inline src/sites/cclw/constants/faqs.tsx:65]: (plural) "partners" refers to single "partner"

File: src/pages/terms-of-use/index.tsx [hunk 2]
```diff
@@ -215,6 +216,78 @@ const TermsOfUse = () => {
...
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://www.ipcc.ch/reports/">IPCC Website</ExternalLink>
+                  </td>
+                  <td>IPCC reports from 2015</td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://www.ipcc.ch/copyright/">View</ExternalLink>
```
Comment (diversemix) [inline src/pages/terms-of-use/index.tsx:288]: This link refers to "This website" - I don't think this statement protects their information on our website @henry-cpr

File: src/pages/terms-of-use/index.tsx [hunk 3]
```diff
@@ -215,6 +216,78 @@ const TermsOfUse = () => {
...
+                <tr>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/topics/global-stocktake/information-portal?field_document_topic_target_id=All&field_document_type_target_id=4174&field_calculated_author_type_target_id=All&field_document_ca_target_id=">
+                      Global Stocktake Information Portal
+                    </ExternalLink>
+                  </td>
+                  <td>Submissions to the first Global Stocktake by parties and non-Party stakeholders </td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
```
Comment (diversemix) [inline src/pages/terms-of-use/index.tsx:228]: This link refers to "This website" - I don't think this statement protects their information on our website @henry-cpr 

Ditto for the rest of the links

File: src/sites/cclw/constants/faqs.tsx [hunk 4]
```diff
@@ -90,29 +103,30 @@ export const FAQS: TFAQ[] = [
...
   {
-    title: "Am I free to download and use your data?",
+    title: "Am I free to download and use the data?",
     content: (
       <>
         <p>
           Yes - and we encourage you to do so! The content of our database is available under the Creative Commons Attribution Licence{" "}
           <ExternalLink url="https://creativecommons.org/licenses/by/4.0/">(CC-BY)</ExternalLink>. Before doing so, you should read our Terms of Use
-          for more information and to find out how to credit our data. You will also need to{" "}
+          for more information and to find out how to cite and credit the resources. If you wish to download the data as a .csv file, please{" "}
```
Comment (diversemix) [inline src/sites/cclw/constants/faqs.tsx:112]: I think we should point out that the terms of use differs for various sources here /cc @henry-cpr

File: src/sites/cclw/constants/faqs.tsx [hunk 5]
```diff
@@ -121,7 +135,94 @@ export const FAQS: TFAQ[] = [
...
+      </>
+    ),
+  },
+  {
+    title: "How do I find Submissions to the UNFCCC that were submitted by multiple Parties?",
+    content: (
+      <>
+        <p>
+          Documents that have been written by multiple Parties, or by one Party on behalf of a coalition/negotiating block, are given the geography
+          profile label "International", which you can search for using the search bar. The specific countries that contributed to the document are
```
Comment (diversemix) [inline src/sites/cclw/constants/faqs.tsx:211]: Pretty sure we have 404'ed this from the geography page.

File: src/sites/cclw/constants/faqs.tsx [hunk 6]
```diff
@@ -273,14 +370,26 @@ export const FAQS: TFAQ[] = [
...
   {
     title: "What are the limitations of our search?",
     content: (
       <>
         <p>
           Our database is not exhaustive. We don’t have access to some law and policy documents, while others aren’t currently machine-readable,
-          meaning we can’t extract the text from them and share them through our resource. We’re working to improve this so that you can search
-          through the full text of more documents in our database. If you think we’re missing some data, you can tell us about it using our{" "}
+          meaning we can’t extract the text from them. We’re working to improve this so that you can search through the full text of more documents in
+          our database. If you think we’re missing some data, you can tell us about it using our{" "}
```
Comment (diversemix) [inline src/sites/cclw/constants/faqs.tsx:392]: Use of "our" inconsistent with the other occurances of this being replaced with "the"

File: src/pages/terms-of-use/index.tsx (hunk 3 above)
Comment (henry-cpr) [reply src/pages/terms-of-use/index.tsx:228]: The relevant paragraph of note for our usage of documents from the link is 
"All official texts, data and documents, including low resolution webcast transmissions, are in the public domain and may be freely downloaded, copied and printed provided no change to the content is introduced, and the source is duly acknowledged."

File: src/sites/cclw/constants/faqs.tsx (hunk 4 above)
Comment (henry-cpr) [reply src/sites/cclw/constants/faqs.tsx:112]: Yes  I think you're right

File: src/pages/terms-of-use/index.tsx (hunk 3 above)
Comment (diversemix) [reply src/pages/terms-of-use/index.tsx:228]: Closed https://linear.app/climate-policy-radar/issue/SOF-55/review-the-use-of-terms-and-conditions-of-third-party-sites

File: src/sites/cclw/constants/faqs.tsx (hunk 4 above)
Comment (diversemix) [reply src/sites/cclw/constants/faqs.tsx:112]: Raised  https://linear.app/climate-policy-radar/issue/SOF-55/review-the-use-of-terms-and-conditions-of-third-party-sites

File: src/pages/terms-of-use/index.tsx (hunk 2 above)
Comment (diversemix) [reply src/pages/terms-of-use/index.tsx:288]: Raised https://linear.app/climate-policy-radar/issue/SOF-55/review-the-use-of-terms-and-conditions-of-third-party-sites

Comment (henry-cpr) [reply src/pages/terms-of-use/index.tsx:288]: It does as it is the license under which we are republishing information from their website, so happy to leave for the moment. Agree not all the links are relevant though, I will need to explore with policy team so accepting for now and then we rework -- as we have the unfccc terms first which covers the public domain license for all the unfccc documents I'm happy we're covered

**Pull request merged**